### PR TITLE
Vernam Cipher in C #2663

### LINKED ISSUE #2663: Vernam cipher implementation in C
## 🚀 Feature

Through this code, I will be implementing Vernam cipher in the C language. It is a symmetric key cryptographic algorithm in which the size of the plain-text and the key is the same.

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

## Motivation

Vernam cipher also known as a one-time pad follows the principle of perfect secrecy. I would like to implement this in the C language.

## Pitch

Please assign this to me under GSSOC'21.

### PR DESCRIPTION
**Pull request by nidh-ish**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Implemented Vernam cipher in C language.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue #2663

Fixes #2663

### CODE REVIEW

**Comment by nidh-ish** [comment]
@bhav09 Why is this one check failing?

File: C/cryptography/Vernam_Cipher.c [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+	}
+
+	else {
+		printf("\nPlease enter a valid choice.\n");
+	}
+
+	}
+
+	return 0;
+}
```
Comment (bhav09) [inline C/cryptography/Vernam_Cipher.c:225]: 1. Need to write an I/O test case.
2.  Mention Time Complexity
3.  Add an extra line at the end of this file

File: C/cryptography/Vernam_Cipher.c [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+		cipher = cipher %  26;
+
+		cipher += 65;
+
+		char enc = cipher;
+
+		encrypted[i] = enc;
+	}
+
+	return encrypted;               //Return the decrypted message
```
Comment (bhav09) [inline C/cryptography/Vernam_Cipher.c]: Add comments to code segments. The comments should be above the segment and not alongside. 
Your code lacks comments. Please do the needful.

File: C/cryptography/Vernam_Cipher.c [hunk 3]
```diff
@@ -0,0 +1,179 @@
+/*In Vernam cipher(also known as the One-Time Pad), the length of the plaintext, ciphertext, and key is the same.
```
Comment (bhav09) [inline C/cryptography/Vernam_Cipher.c:1]: Can you please brief a little about ciphertext in a few lines? That would be helpful.

**Review by bhav09 — COMMENTED** [review]
Have reviewed it. Do check the comments and modify the file accordingly :)

**Comment by nidh-ish** [comment]
@bhav09 I did what you said. Still getting the same error.

**Comment by bhav09** [comment]
Ignore those erroneous checks as of now.

**Pull request merged**